.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+const talendEslintRcPath = require.resolve('@talend/scripts-config-eslint/.eslintrc');
```

Comment:
Used the linter to verify if upgrades where breaking things or not. Updated a little the config at the same time.